solidity/core/test/validator-manager/outboxMultisigValidatorManager.test.ts

```diff
@@ -0,0 +1,116 @@
+import { expect } from 'chai';
+import { ethers } from 'hardhat';
+import { types, utils, Validator } from '@abacus-network/utils';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+
+import {
+  Outbox,
+  Outbox__factory,
+  OutboxMultisigValidatorManager,
+  OutboxMultisigValidatorManager__factory,
+} from '../../types';
+import { signCheckpoint } from './utils';
+
+const OUTBOX_DOMAIN = 1234;
+const INBOX_DOMAIN = 4321;
+const QUORUM_THRESHOLD = 2;
+
+describe('OutboxMultisigValidatorManager', () => {
+  let validatorManager: OutboxMultisigValidatorManager,
+    outbox: Outbox,
+    signer: SignerWithAddress,
+    validator0: Validator,
+    validator1: Validator;
+
+  before(async () => {
+    const signers = await ethers.getSigners();
+    [signer] = signers;
+    const [, validatorSigner0, validatorSigner1] = signers;
+    validator0 = await Validator.fromSigner(validatorSigner0, OUTBOX_DOMAIN);
+    validator1 = await Validator.fromSigner(validatorSigner1, OUTBOX_DOMAIN);
+  });
+
+  beforeEach(async () => {
+    const validatorManagerFactory = new OutboxMultisigValidatorManager__factory(
+      signer,
+    );
+    validatorManager = await validatorManagerFactory.deploy(
+      OUTBOX_DOMAIN,
+      [validator0.address, validator1.address],
+      QUORUM_THRESHOLD,
+    );
+
+    const outboxFactory = new Outbox__factory(signer);
+    outbox = await outboxFactory.deploy(OUTBOX_DOMAIN);
+    await outbox.initialize(validatorManager.address);
+  });
+
+  describe('#improperCheckpoint', () => {
+    const root = ethers.utils.formatBytes32String('test root');
+    const index = 1;
+
+    it('accepts an improper checkpoint if there is a quorum', async () => {
+      const signatures = await signCheckpoint(
+        root,
+        index,
+        [validator0, validator1], // 2/2 signers, making a quorum
+      );
+
+      // Send message with signer address as msg.sender
```

Comment:
Why call this out in a comment? It doesn't matter what address this tx is sent from, does it?

Reply:
yeah this is just a relic of the old validator manager test I copied, removed